joern-cli/frontends/rubysrc2cpg/src/test/scala/io/joern/rubysrc2cpg/querying/AttributeAccessorTests.scala

```diff
@@ -0,0 +1,50 @@
+package io.joern.rubysrc2cpg.querying
+
+import io.joern.x2cpg.Defines
+import io.joern.rubysrc2cpg.testfixtures.RubyCode2CpgFixture
+import io.shiftleft.codepropertygraph.generated.{Operators, DispatchTypes}
+import io.shiftleft.codepropertygraph.generated.nodes.{Call, FieldIdentifier, Identifier}
+import io.shiftleft.semanticcpg.language.*
+
+class AttributeAccessorTests extends RubyCode2CpgFixture {
+
+  "`x.y=1` is approximated by a `x.y =` assignment with argument `1`" in {
+    val cpg = code("""x = Foo.new
+                     |x.y = 1
+                     |""".stripMargin)
+    inside(cpg.assignment.where(_.source.isLiteral.codeExact("1")).l) {
+      case xyAssign :: Nil =>
+        xyAssign.lineNumber shouldBe Some(2)
+        xyAssign.code shouldBe "x.y = 1"
+
+        val fieldTarget = xyAssign.target.asInstanceOf[Call]
+        fieldTarget.code shouldBe "x.@y"
+        fieldTarget.name shouldBe Operators.fieldAccess
+        fieldTarget.methodFullName shouldBe Operators.fieldAccess
+        fieldTarget.dispatchType shouldBe DispatchTypes.STATIC_DISPATCH
+
+        inside(fieldTarget.argument.l) {
+          case (base: Identifier) :: (field: FieldIdentifier) :: Nil =>
+            base.name shouldBe "x"
+            field.canonicalName shouldBe "@y"
```

Comment:
Please add a check for `field.name shouldBe "y"`.

Reply:
There is no such field, but I will make this the case for `.code`